codemon/CodemonListen.py

```diff
@@ -38,7 +38,10 @@ def isModified(event):
   if filename != foldername and filename != "prog" and filename != "input.txt": 
     print(colored.yellow('\nChange made at '+ filename))
     print(colored.cyan('\nCompiling '+ filename))
-    if os.system('g++ ' + filename + ' -o ' + 'prog') == 0:
+    folder,lang = filename.split('.')
+    fileLocation = folder + '/' + filename

```

Comment:
rename `fileLocation` to `targetFile`, rename `inputLocation` to `targetInput` or simply `input`